src/RobotPool.cpp

```diff
@@ -143,23 +143,22 @@ bool RobotPool::robotAvailable(RobotType _type)
 	switch (_type)
 	{
 	case ROBOT_DIGGER:
+	{
 		return getDigger() != nullptr;
-		break;
-
+	}
 	case ROBOT_DOZER:
-		return getDozer() != nullptr;
-		break;
-
+    {
+        return getDozer() != nullptr;
+    }
 	case ROBOT_MINER:
```

Comment:
Hmm, indentation looks a bit off in this section.

Reply:
(╯°□°)╯︵ ┻━┻ clang-format!

Reply:
"Fixed"

Reply:
Check your diffs when you commit :wink:

And maybe disable clang-format.

Reply:
I do, I can't see the difference between tabs and spaces in the diffs. Since I have tabs stops set to four, spaces and tabs look the same on my machine vs github and the diff coloring is just a solid bar of color, no whitespace markers.

See: https://stackoverflow.com/a/52764248/421178 for an example of what GitKraken shows me.